src/components/task/index.cjsx

```diff
@@ -65,15 +68,33 @@ module.exports = React.createClass
 
     else if @state.currentStep is -1
       if allStepsCompleted
-        footer = <Router.Link to="dashboard" className="btn btn-primary">Back to Dashboard</Router.Link>
-
-        <div className="task task-completed">
-          {breadcrumbs}
-          <BS.Panel bsStyle="default" footer={footer}>
-            <h1>You Are Done.</h1>
-            <h3>Great Job!</h3>
-          </BS.Panel>
-        </div>
+        if model.type is 'practice'
```

Comment:
Given the amount of additional logic for a `practice` task, maybe this component should be split by type (similar to `../task-step/index` and dispatch to a `reading.cjsx` or a `practice.cjsx`. If there's enough combined logic then maybe abstract into a Mixin?